src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -555,6 +580,46 @@ PyChipError pychip_DeviceController_SetCheckMatchingFabric(bool check)
     return ToPyChipError(CHIP_NO_ERROR);
 }
 
+PyChipError pychip_DeviceController_SetIcdRegistrationParameters(chip::Controller::DeviceCommissioner * devCtrl, bool enabled,
```

Comment:
Can we use a struct instead to reduce the number of arguments to register?

Reply:
updated.